### ISSUE TITLE
TLJH + NBgrader: No Assignment or Formgrader buttons 

### ISSUE THREAD

**Issue by TRM13** [jupyter/nbgrader#1325]
I have setup TLJH on AWS with JupyterLab as the interface. The installation of nbgrader went fine but I don't have any "Assignment" or "Formgrader" buttons when I log into the web site as instructor.

### Operating system
AWS - Ubuntu 18.04 fully patched
### `nbgrader --version`
[GCC 8.3.0]
nbgrader version 0.6.1
### `jupyterhub --version` (if used with JupyterHub)
1.1.0
### `jupyter notebook --version` 
6.0.3
(and  jupyterlab-2.1.0 jupyterlab-server-1.1.0)
### Expected behavior
When I login as instructor to the amazonaws.com/hub/user/instructor/tree I should see the assignment or formgrader buttons
### Actual behavior
I don't see the assignment and formgrader buttons on the web page

**Comment by TRM13** [comment]
And of course AFTER I posted this I found .....
https://github.com/jupyter/nbgrader/issues/1006
Sigh. As soon as you hit that comment or send button you find the answer. Not the answer I was looking for as I am whole heartedly in love with the Lab interface but it is now pushed out to 0.7 release :(
Thanks for all the work on nbgrader everyone and I've signed up to 1006 so I'll get notices when it does happen. TTYL
